webapp/src/features/intersections/map/control-panel.tsx

```diff
@@ -730,8 +428,342 @@ function ControlPanel() {
+                Download All Message Data
+              </Button>
+              <Box sx={{ display: 'flex', justifyContent: 'flex-start', alignItems: 'center' }}>
+                <Typography fontSize="medium" sx={{ mt: 1, mr: 1 }}>
+                  Upload Message Data:
+                </Typography>
+                <Button variant="text" component="label" sx={{ mt: 1 }} startIcon={<UploadFile color="primary" />}>
+                  <VisuallyHiddenInput
+                    accept=".zip"
+                    id="upload"
+                    name="upload"
+                    type="file"
+                    multiple={false}
+                    onChange={(e: ChangeEvent<HTMLInputElement>) => {
+                      openMessageData(e.target.files)
+                    }}
+                  />
+                  Choose File
+                </Button>
+              </Box>
+            </div>
+          </AccordionDetails>
+        </Accordion>
+
+        <Accordion
+          disableGutters
+          expanded={isExpandedDecoder}
+          onChange={() => setIsExpandedDecoder(!isExpandedDecoder)}
+          sx={{
+            py: 0.5,
+            borderRadius: '4px',
+          }}
+        >
+          <AccordionSummary expandIcon={<ExpandMoreOutlined />}>
+            <Typography variant="h6">
+              ASN.1 Decoding{' '}
+              {decoderModeEnabled && (
+                <Chip
+                  label="Decoder Mode Active"
+                  className="blink_me"
+                  sx={{ ml: 2, borderRadius: '4px' }}
+                  color="warning"
+                />
+              )}
+            </Typography>
+          </AccordionSummary>
+          <AccordionDetails>
+            <div
+              className="control-panel"
+              style={{
+                padding: '10px 30px 0px 20px',
+              }}
+            >
+              <div>
+                <Typography sx={{ m: 1 }} color="white">
+                  <Typography sx={{ textIndent: '0px each-line' }} color="white">
+                    This tool allows you to decode and render ASN.1 encoded data. To use this tool:
+                  </Typography>
+                  <br />
+                  <Typography sx={{ textIndent: '-15px' }} color="white">
+                    1. Enable Decoder Mode. This will disable all other map data until Decoder Mode is disabled.
+                  </Typography>
+                  <br />
+                  <Typography sx={{ textIndent: '-15px' }} color="white">
+                    2. Hit "Decoder + Render Data" to open the decoder dialog, where you can enter/upload asn.1 encoded
+                  </Typography>
+                  <br />
+                  <Typography sx={{ textIndent: '-15px' }} color="white">
+                    3. Return to the map to render your data. Data can be toggled on/off in the dialog menu.
+                  </Typography>
+                </Typography>
```

Comment:
Remove `<br />` elements, remove indentations, remove text color
```suggestion
                <Typography>
                  This tool allows you to decode and render ASN.1 encoded data. To use this tool:
                </Typography>
                <Typography>
                  1. Enable Decoder Mode. This will disable all other map data until Decoder Mode is disabled.
                </Typography>
                <Typography>
                  2. Hit "Decoder + Render Data" to open the decoder dialog, where you can enter/upload asn.1 encoded
                </Typography>
                <Typography>
                  3. Return to the map to render your data. Data can be toggled on/off in the dialog menu.
                </Typography>
```